coralproject/talk
# [CORL-1327] allow configuration of admin and moderator comment tags

## What does this PR do?

Adds new more specific tags to comments based on user role: instead of tagging all comments from staff, moderator, and admin users as "staff", they are now tagged with their respective roles. Also allows customization of display labels for these tags, defaulting all three to "staff". 

## What changes to the GraphQL/Database Schema does this PR introduce?

- Adds `ADMIN` and `MODERATOR` tags to `Tag` enum
- adds `staff.adminLabel` and `staff.moderatorLabel` to tenant settings 

## How do I test this PR?

- under general configuration, change the labels for moderator or admin roles to something other than "staff"
- make a comment as a moderator or admin
- observe the comment being tagged with the appropriate role
